docker/build/build-deps.sh

```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2023 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+set -euo pipefail
+
+source "$HOME/.cargo/env"
+# Disable usage of instructions from the ADX extension to avoid incompatibility
+# with old CPUs, see https://gitlab.com/dannywillems/ocaml-bls12-381/-/merge_requests/135/
```

Comment:
This comment is misplaced, it's setting the `BLST_PORTABLE` what's written here possible